[2023-10-20 23:28:04,755] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-08-17T00:00:00+00:00 [queued]>
[2023-10-20 23:28:04,774] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-08-17T00:00:00+00:00 [queued]>
[2023-10-20 23:28:04,775] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:28:04,777] {taskinstance.py:1377} INFO - Starting attempt 2 of 6
[2023-10-20 23:28:04,778] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:28:04,809] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-08-17 00:00:00+00:00
[2023-10-20 23:28:04,817] {standard_task_runner.py:52} INFO - Started process 212 to run task
[2023-10-20 23:28:04,823] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-08-17T00:00:00+00:00', '--job-id', '1162', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpn0j27puu', '--error-file', '/tmp/tmpfkenkfap']
[2023-10-20 23:28:04,828] {standard_task_runner.py:80} INFO - Job 1162: Subtask segunda_tarea
[2023-10-20 23:28:04,978] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-08-17T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:28:05,137] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-08-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-17T00:00:00+00:00
[2023-10-20 23:28:05,139] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:28:05,140] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:28:05,153] {subprocess.py:85} INFO - Output:
[2023-10-20 23:28:05,156] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:28:05,157] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:28:05,241] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230817T000000, start_date=20231020T232804, end_date=20231020T232805
[2023-10-20 23:28:05,282] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:28:05,340] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
